citus-ha.spec

```diff
@@ -8,11 +8,11 @@ Summary:	Auto-HA support for Citus
 Name:		%{sname}%{?pkginfix}_%{pgmajorversion}
 Provides:	%{sname}_%{pgmajorversion}
 Conflicts:	%{sname}_%{pgmajorversion}
-Version:	2.0.0
+Version:	2.1.0
 Release:	1%{dist}
 License:	AGPLv3
```

Comment:
Nitpick, but this is not correct (not that it matters for now, but this should read `Proprietary`)

Reply:
good catch will change